auth/middleware/passport.ts

```diff
@@ -0,0 +1,30 @@
+import { Strategy as JwtStrategy, ExtractJwt } from "passport-jwt";
+import { getRepository } from "typeorm";
+import jwtConfig from "../configurations/jwt.js";
+import User from "../models/user";
+import { to } from "../services/Util";
+
+module.exports = function (passport: any) {
+  passport.use(
+    new JwtStrategy(
+      {
+        jwtFromRequest: ExtractJwt.fromAuthHeaderAsBearerToken(),
+        secretOrKey: jwtConfig.encryption
+      },
+      async function (jwtPayload: any, done: any) {
+        let err;
+        let user;
+        const userRepository = await getRepository(User);
+        [err, user] = await to(userRepository.findOne(jwtPayload.id));
+        if (err) {
+          return done(err, false);
+        }
+        if (user) {
+          return done(null, user);
+        } else {
+          return done(null, false);
```

Comment:
should probably return error message here

Reply:
DB errors are handled in the above if condition.
Any other error related to JWT is handled by the passport node module